### DISCUSSION TITLE
Project health

### DISCUSSION THREAD

**Discussion by chewstiltzoo** [jpanther/congo#917] [General]
Is the project still alive? It has been three months since the last commit and almost four months since the last release. I hope that @jpanther is all right and work on this great project continues.

**Comment by jpanther** [comment]
I'm here! Sorry, it's been a very busy time for me. The project is still very much active. I'll try to push some updates out this week. I almost had a stable release ready to go but I'll merge in some of the dependency updates before making a new release.

**Reply by chewstiltzoo** [reply]
That’s great! I’m sorry, I didn’t mean to pressure. The project is in great shape as it is. I’m glad to hear you’re OK and that work on this superb theme continues in due time. Thank you for all the great work you put into it.
